v2/pkg/core/workflow_execute.go

```diff
@@ -18,7 +20,7 @@ func (e *Engine) executeWorkflow(input string, w *workflows.Workflow) bool {
 		swg.Add()
 		func(template *workflows.WorkflowTemplate) {
 			if err := e.runWorkflowStep(template, input, results, &swg, w); err != nil {
-				gologger.Warning().Msgf("[%s] Could not execute workflow step: %s\n", template.Template, err)
+				gologger.Warning().Msgf(couldNotExeWorkflow, template.Template, err)
```

Comment:
I would recommend using **UPPER_SNAKE_CASE** for constants. To make it clear what value it holds, I'd include the `MESSAGE_TEMPLATE` suffix. E.g. `WORKFLOW_EXECUTION_ERROR_MSG_TEMPLATE` or something.

It might require changing some `gofmt` rules though.

Reference: https://github.com/Pungyeon/clean-go-article

CC @Ice3man543, @Mzack9999

Reply:
Correction: fully upper-case style should only be used for exported constants

Reply:
Done